Specs/Scene/BatchTableHierarchySpec.js

```diff
@@ -164,43 +223,49 @@ describe("Scene/BatchTableHierarchy", function () {
   });
 
   it("hasProperty returns true if the feature has this property", function () {
+    const deep = true;
     const hierarchy = new BatchTableHierarchy({
-      extension: hierarchyExtension,
+      extension: clone(hierarchyExtension, deep),
```

Comment:
is there a reason we're cloning the hierarchy extensions for the `hasProperty` and `getProperty` tests?

Reply:
Yes, the batch table hierarchy code was modifying the input in place, so running multiple tests was causing issues.

Reply:
I get that, I just thought that the `setProperty` tests were already cloning for themselves. but it's fine, this isn't that important